tests/format/handlebars/single-quote-option/attributes.hbs

```diff
@@ -1,8 +1,12 @@
 <div title='My title'></div>
 <div title="My title"></div>
 <div title='My "title"'></div>
+<div title="My 'title'"></div>
```

Comment:
👍  then, we can remove the test case on line 5 which is a duplicate

Reply:
Removed line 4 because 5 was there before.